### DISCUSSION TITLE
Indicate which translation strings are in use

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17131] [Feature Requests]
Would be nice to see which translation strings are in use, in the translation string list view (´/admin/settings/translation-strings`).
Plus warn the user while deleting a translation string, that is in use.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
